app/components/pages/DashboardPage.js

```diff
@@ -238,7 +238,7 @@ export default function DashboardPage({ session }) {
         <div className="w-full h-full flex flex-col gap-y-5 p-10 pt-20 overflow-y-scroll">
           {/* TODO: This page should be under app/page.js probably */}
           <PageTitle title="Dashboard" />
-          <div className="h-full flex flex-row gap-x-5">
+          <div className="h-full flex md:flex-row sm:flex-col gap-x-5 sm:gap-y-5">
```

Comment:
to target sm screens, i think you don't need to add sm: before it - adding sm: would only target screens equal or greater than sm size (https://tailwindcss.com/docs/responsive-design#working-mobile-first)

Reply:
I added it because I want a particular size for screens between sm and md